File: main/java/com/andr/testlayout/MainActivity.java
package com.andr.testlayout;

import android.content.Intent;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.fragment_voice_to_text);
        Fragment basicChatList = VoiceToTextFragment.newInstance("","");
        getSupportFragmentManager().beginTransaction().replace(R.id.frag_parentLayout,basicChatList).commit();
    }
    @Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {
        // Override this method in the activity that hosts the Fragment and call super
        // in order to receive the result inside onActivityResult from the fragment.
        super.onActivityResult(requestCode, resultCode, data);
    }
}

File: KotlinTemplates/app/src/main/java/com/kotlintemplates/MainActivity.kt
package com.kotlintemplates

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.kotlintemplates.Launcher.View.TemplateListFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        launchInitialFragment()
    }


    private fun launchInitialFragment(){
        supportFragmentManager.beginTransaction().replace(R.id.container_templates, TemplateListFragment()).commit()
    }
}
